specs/Menu.md

```diff
@@ -1,731 +1,540 @@
 
-Menus are often combined with a popover or similar floating component so the menu appears over other items and is subsequently dismissed.
+## Prior art
 
-### MenuItem
+As a part of the spec definitions in Fluent UI, a research effort has been made through [Open UI](https://open-ui.org/). The current research proposal is available as an open source contribution undergoing review ([LINK to research proposal](https://github.com/WICG/open-ui/pull/249))
 
-Most menus have a concept of a specific menuitem. It is an integral part of the menu and will have some of its own sections throughout.
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
 
-## References
+All mentions of v7 or v8 == `@fluentui/react`
+All mentions of v0 == `@fluentui/react-northstar`
 
-[CodeSandbox of implemented versions](https://codesandbox.io/s/menuexamples-uybsr)
+The most relevant comparison that can be achieved between the two libraries is between `ContextualMenu` in v7 and a combination of `Menu`, `Popup` and `ToolbarItem` in v0.
 
-- [Aria practices](https://www.w3.org/TR/wai-aria-1.1/#menu)
-- [Fabric](https://developer.microsoft.com/en-us/fabric#/controls/web/contextualmenu)
-- [Stardust](https://microsoft.github.io/fluent-ui-react/components/menu/definition)
-- [Material-ui](https://material-ui.com/components/menus/)
-- [Chakra-ui](https://chakra-ui.com/menu)
-- [AntDesign](https://ant.design/components/menu/)
-- [Jquery-ui](https://jqueryui.com/menu/)
-- [Carbon-ui](https://www.carbondesignsystem.com/components/overflow-menu/code)
-- [Fast-DNA](https://github.com/microsoft/fast-dna/blob/master/packages/fast-components-react-base/src/context-menu/context-menu.tsx)
-- [BluePrintJs](https://blueprintjs.com/docs/#core/components/menu)
+v0 suffers from a consistency issue that the control used in `Menu` and the menu variant of `ToolbarItem` are not actually the same component and have different behaviour. However, semantically for the purposes of this spec, they represet the same control that will be implemented.
 
-### Notes from references:
+Note that the below code samples are not meant to be complete, but to highlight differences between the two libraries. Please refer to official docsites for actual API references.
 
-- Most render children, rather than passing a list of items in as props
-- Menu is usually a `<ul>` with `<li>` as children
-- Some are popout menus rather than just being a list, but most do not have an explicit popout.
-- Most have a concept of submenus, either explicitly or as a result of nested lists.
-- The line between list and menu is fuzzy.
+### Positioning
 
-## API
+`ContextualMenu` in v7 is a component that also exposes the API to control the positioning of the temporary popup surface that the Menu is rendered on. This aspect of the v7 component should be compared with the `Popup` component in v0 since the v0 `Menu` is created as a standalone component with no positioning properties.
```

Comment:
> since the v0 `Menu` is created as a standalone component with no positioning properties.

We have positioning properties for submenus...

Reply:
Aaaaah that makes my life complicated. I will resolve this for now and track this issue in the checklist in PR description